src/Interactive/Features/Interactive/Core/InteractiveHost.Service.cs

```diff
@@ -276,9 +284,25 @@ private static string GenerateUniqueChannelLocalName()
 
                 lock (_sessionGuard)
```

Comment:
@cston, is the lock actually protecting reads and writes to ```_lastTask```?  If so, should it be renamed?

Also, do we want to use something like ```SemaphoreSlim.WaitAsync``` so that tasks can yield while they wait for their turn to add to the "queue"?